File: jsonExample/src/main/java/com/wujie/jsonexample/MyApplication.java
package com.wujie.jsonexample;

import android.app.Application;

import io.realm.Realm;

/**
 * Created by wujie on 2018/7/16/016.
 */

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
    }
}

File: kotlinExample/src/main/java/com/wujie/kotlinexample/MyApplication.kt
package com.wujie.kotlinexample

import android.app.Application
import io.realm.Realm

/**
 * Created by wujie on 2018/7/18/018.
 */
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
    }
}